student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
```

Comment:
- [ ] This inheritance will not work. Kindly import the **`Person`** class into this file.

Reply:
Thank you for pointing out that, but I think this is not mandatory because you will need to import anything you are going to use in the `irb `terminal. But I am going to fix it anyway. @julie-ify